File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):    
    even = 0;
    odd = 0;    
    answer = [even, odd]
    
    for i in num_list :
        if i % 2 == 0 :
            answer[0] += 1
        else :
            answer[1] += 1
            
    return answer

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [];
    answer[0] = 0;
    answer[1] = 0;
    for(let i of num_list){
        if(i % 2 == 0){
            answer[0] += 1;
        }else{
            answer[1] += 1;
        }
    }
    return answer;
}